CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 =========
 
+Version 0.2.6
```

Comment:
Do you not increase the minor version when you add functionality, leaving patch version for bug fixes and really minor things? This would mean a version 0.3.0. OK, not much content in this comment :D.

Reply:
Hum I forgot how this works actually, if you think this deserves a version 0.3.0 than I am up for it.

Reply:
No big deal.